### DISCUSSION TITLE
Contact with translators

### DISCUSSION THREAD

**Discussion by SabinaChita** [hedyorg/hedy#4116] [Announcements]
Symposium, email, online meeting?

**Comment by Felienne** [comment]
Hi @SabinaChita!

I made this issue into a Discussion. Issues are great for things that can be picked up immediately; Discussions are better for when we are still debating what we want to do.

Maybe people can even chip in here with ideas about how to foster the translation community. A few issues are:

* Translators are sometimes "drive by": They translate once and then disappear.
* We cannot contact translators, since they have no GitHub account (only a Weblate one) or they do not reply to GitHub notifications

These are maybe symptoms of the fact that translators (other than teachers and programmers) do not feel part of the "Hedy family" yet.
